platform/javascript/detect.py

```diff
@@ -130,7 +130,6 @@ def configure(env):
         env.Append(CCFLAGS=["-fsanitize=leak"])
         env.Append(LINKFLAGS=["-fsanitize=leak"])
     if env["use_safe_heap"]:
-        env.Append(CCFLAGS=["-s", "SAFE_HEAP=1"])
```

Comment:
Why remove this?

Reply:
See comment: `Also fix a warning about SAFE_HEAP being a linker only flag.`

Reply:
Do you have a source which can verify the warning? I don't want it to bite back later.

Reply:
Do you have a source which can verify that warning? I don't want it to bite back later.